### DISCUSSION TITLE
PG_NET follow redirect?

### DISCUSSION THREAD

**Discussion by northfacejmb** [supabase/supabase#22491] [Questions]
I'm working on a job_que system using PG_NET and it has been working great, but now I need it to call a new endpoint that has a 308 redirect. Is there a way to enable PG_NET to automatically follow redirects?

**Comment by GaryAustin1** [comment]
I'm not sure, but:
https://github.com/supabase/pg_net/issues/87
https://github.com/supabase/pg_net/pull/113
Seems like support was added.

**Reply by northfacejmb** [reply]
thank you! after digging into it more I realized it was partially due to sending the request to http when it should have been https
